lms/templates/student_account/login.underscore

```diff
@@ -1,13 +1,6 @@
 <div class="js-form-feedback" aria-live="assertive" tabindex="-1">
 </div>
 
-<% if ( context.createAccountOption !== false && !context.syncLearnerProfileData && !(context.enterpriseName && context.currentProvider) ) { %>
```

Comment:
Interesting -- it looks like edX already hides this for the "enterprise" context. 

But, since I don't know how to set up all that stuff, it seems easy enough to just remove it from the template. 

I wonder what `syncLearnerProfileData` is... ?